Epsilon.Host.Frontend/src/components/TermDateFilter.vue

```diff
@@ -0,0 +1,173 @@
+<template>
+	<button class="refresh-button" @click="resetDates">↻</button>
+	<div ref="dateRangeMenu" class="date-range-menu">
+		<button @click="toggleMenu">Date range</button>
+		<div class="menu-content" :class="{ 'menu-visible': menuVisible }">
+			<div class="date-range-filter">
+				<div class="date-input">
+					<label for="startDate">Start date:</label>
+					<input
+						id="startDate"
+						v-model="startDate"
+						type="date"
+						@input="filterItems" />
+				</div>
+				<div class="date-input">
+					<label for="endDate">End date:</label>
+					<input
+						id="endDate"
+						v-model="endDate"
+						type="date"
+						@input="filterItems" />
+				</div>
+			</div>
+		</div>
+	</div>
+</template>
+
+<script lang="ts" setup>
+import { ref, onMounted, defineProps } from "vue"
+import { useStore } from "vuex"
+const store = useStore()
+
+const props = defineProps<{
+	items: Array<{ start_at: string; end_at: string }> | null
```

Comment:
Please create an interface. 
`interface TermSelection{
start_at: string
end_at: string
}`

Then mention the time like this:

```suggestion
	items: TermSelection[] | null
```

Reply:
Should the interface be defined under Epsilon.Abstractions in the services folder? Or is there another designated place for these kinds of interfaces?